app/controllers/api/v2/job_invocations_controller.rb

```diff
@@ -70,14 +70,10 @@ def show
       api :POST, '/job_invocations/', N_('Create a job invocation')
       param_group :job_invocation, :as => :create
       def create
-        if job_invocation_params[:feature].present?
-          composer = composer_for_feature
```

Comment:
The composer was, among other things, setting `remote_execution_feature_id` on the `job_invocation` being composed. This no longer happens

Reply:
I believe that's fixed now